Bug in uart binary client

**Describe the bug**

On running  `make uart_binary_client` the build system defaults to errors.

```
UART client, runs in background
** UART client, keep running in separate shell
Listening for logs on port /dev/ttyACM0, and using files in "/home/anne/workspace/bluenet/source" to find the log formats.
Traceback (most recent call last):
  File "/home/anne/workspace/bluenet/scripts/log-client.py", line 44, in <module>
    bluenetLogs.setSourceFilesDir(sourceFilesDir)
  File "/home/anne/.local/lib/python3.8/site-packages/bluenet_logs/BluenetLogs.py", line 20, in setSourceFilesDir
    self.logLineRetriever.setSourceFilesDir(dir)
  File "/home/anne/.local/lib/python3.8/site-packages/bluenet_logs/LogLineRetriever.py", line 39, in setSourceFilesDir
    self._cacheFileContents()
  File "/home/anne/.local/lib/python3.8/site-packages/bluenet_logs/LogLineRetriever.py", line 82, in _cacheFileContents
    lines = file.readlines()
  File "/usr/lib/python3.8/codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xd3 in position 7734: invalid continuation byte
make[3]: *** [CMakeFiles/uart_binary_client.dir/build.make:59: CMakeFiles/uart_binary_client] Error 123
make[2]: *** [CMakeFiles/Makefile2:272: CMakeFiles/uart_binary_client.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:279: CMakeFiles/uart_binary_client.dir/rule] Error 2
make: *** [Makefile:229: uart_binary_client] Error 2
```

**To Reproduce**

Checkout and run  `make uart_binary_client`.

**Expected behavior**

Working binary UART client.

**Already done**

At least on an error like this the user should be hinted in setting the `-DDOWNLOAD_BLUENET_LIB_LOGS=ON` option for cmake. 

```
Requirement already satisfied: bluenet-logs in /home/anne/.local/lib/python3.8/site-packages (0.1.3)
Requirement already satisfied: crownstone-uart~=2.0 in /home/anne/.local/lib/python3.8/site-packages (from bluenet-logs) (2.0.0)
Requirement already satisfied: crownstone-core>=2.0.0 in /home/anne/.local/lib/python3.8/site-packages (from crownstone-uart~=2.0->bluenet-logs) (2.1.0)
Requirement already satisfied: pyserial>=3.4.0 in /home/anne/.local/lib/python3.8/site-packages (from crownstone-uart~=2.0->bluenet-logs) (3.5)
Requirement already satisfied: pyaes~=1.6 in /home/anne/.local/lib/python3.8/site-packages (from crownstone-core>=2.0.0->crownstone-uart~=2.0->bluenet-logs) (1.6.1)
```

Not that that fixed it. :-)

--- a/bluenet_logs/LogLineRetriever.py
+++ b/bluenet_logs/LogLineRetriever.py
@@ -57,11 +57,12 @@ class LogLineRetriever:
 
 	def _cacheFileNames(self):
 		"""
-		Cache all fileNames in sourceFilesDir
+		Cache all C/C++ fileNames in sourceFilesDir
 		"""
 		for root, dirs, files in os.walk(self.sourceFilesDir):
 			for fileName in files:
-				self.fileNames.append(os.path.join(root, fileName))
+				if fileName.endswith((".cpp",".c",".h",".hpp")):
+					self.fileNames.append(os.path.join(root, fileName))
 
 	def _cacheFileNameHashes(self):
 		"""

